tests/performance/C-graphite/CMakeLists.txt

```diff
@@ -8,8 +8,8 @@ FUNCTION(ADD_C_GRAPHITE_TEST TEST_NAME TEST_DIR INPUT_FILE H5_FILE)
   MESSAGE_VERBOSE("Adding test ${TEST_NAME}")
   SET (WDIR "${qmcpack_BINARY_DIR}/tests/performance/C-graphite/${TEST_DIR}")
   SET (SDIR "${qmcpack_SOURCE_DIR}/tests/performance/C-graphite/sample/${TEST_DIR}")
-
-  SET(QMC_APP "${qmcpack_BINARY_DIR}/bin/qmcpack")
+  GET_TARGET_PROPERTY( QMC_APP_NAME qmcpack OUTPUT_NAME )
+  SET ( QMC_APP "${qmcpack_BINARY_DIR}/bin/${QMC_APP_NAME}" )
```

Comment:
I think "${qmcpack_BINARY_DIR}/bin/${QMC_APP_NAME}" can be replaced with generation expression `$<TARGET_FILE:qmcpack>`

Reply:
This is working. Not touching it.

Reply:
The requested change works and make the cmake files sane. Unless touching those lines introduce merge conflicts with upcoming PRs, getting them changed in this PR is still preferred.

Reply:
Investigating

Reply:
If you want this you'll have to push something that works.

Reply:
I changed macro. Please check if it works on your side, if yes, please propagate the change to other places, if no, revert my commit and I will investigate separately.